connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -345,15 +342,24 @@ private void waitForConsumerGroupOffsetSync(Consumer<byte[], byte[]> consumer, L
         }, OFFSET_SYNC_DURATION_MS, "Consumer group offset sync is not complete in time");
     }
 
+    private void waitForConsumingAllRecords(Consumer<byte[], byte[]> consumer) throws InterruptedException {
+        final AtomicInteger totalConsumedRecords = new AtomicInteger(0);
+        waitForCondition(() -> {
+            ConsumerRecords records = consumer.poll(Duration.ofMillis(500));
```

Comment:
Can we add the types `<byte[], byte[]>` to `ConsumerRecords`?

Reply:
Done. Thanks.